File: data/src/main/java/org/oppia/data/backends/gae/model/ConceptCard.java
package org.oppia.data.backends.gae.model;

import com.squareup.moshi.Json;

import java.util.List;

public class ConceptCard {

  /*
   * Parent Hierarchy: ConceptCard
   */

  //TODO: ImmutableList<> and List<> error
  //ImmutableList<> does not work but List<> works

  @Json(name = "concept_card_dicts")
  public List<SkillContents> conceptCardDicts;

}

File: data/src/main/java/org/oppia/data/backends/gae/model/ConceptCard.kt
package org.oppia.data.backends.gae.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ConceptCard(

  @Json(name = "concept_card_dicts") val conceptCardDicts: List<SkillContents>?

)
